Pipfile

```diff
@@ -4,7 +4,8 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-pyramid-heroku = {editable = true, path = "."}
+pyramid-heroku = {editable = true,path = "."}
+mypy = "*"
```

Comment:
Why is mypy listed here? It should be enough to have it only under `[dev-packages]`.

Reply:
Yup. I didn't realize I added it at two places. Removing.